src/plugins/workflow_designer/src/library/DocWorkers.cpp

```diff
@@ -710,6 +710,8 @@ void SeqWriter::data2doc(Document *doc, const QVariantMap &data) {
         RawSeqWriter::data2document(doc, data, context);
     } else if (BaseDocumentFormats::GFF == fid) {
         GFFWriter::data2document(doc, data, context);
+    }else if (BaseDocumentFormats::UGENEDB == fid) {
```

Comment:
Do not use yoda style in new code (move the constant/expected value to the right)